src/OpenTelemetry/Trace/BatchExportActivityProcessorOptions.cs

```diff
@@ -0,0 +1,91 @@
+// <copyright file="BatchExportActivityProcessorOptions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Diagnostics;
+using System.Security;
+using OpenTelemetry.Internal;
+
+namespace OpenTelemetry.Trace
+{
+    public class BatchExportActivityProcessorOptions : BatchExportProcessorOptions<Activity>
```

Comment:
Just throwing this out there, I don't think this _needs_ to be public. Since what is exposed in the exporter options is `BatchExportProcessorOptions<Activity>`. Reason to make it internal sealed is greater flexibility to change it in the future if needed 😄

Reply:
I made it `public` so that it can be reused in contrib and other repositories as well.

Reply:
That's a good point, I don't feel strongly either way. Historically we have duplicated stuff in contrib with a preference to keeping the public API minimal and the "forever-support" footprint small. Defer to @cijothomas!

If we do keep it public here, probably should add some /// comments. Maybe also mark it sealed?

Reply:
Okay either way - as this is doing things from spec, okay to merge as is, keeping public.

Reply:
agree with making sealed.
(Will merge without it! and can do a small followup for it.. I have clicked "merge main" for this PR quite a lot but always misses the chance to merge before another PR goes in!)

Reply:
@cijothomas Just to double-check. You prefer to make it sealed - right?

Reply:
yes

Reply:
Trying to address it here: https://github.com/open-telemetry/opentelemetry-dotnet/pull/2350